malaria-ios/ViewControllers/Take Pills Screen and Pills Stats/PagesManagerViewController.swift

```diff
@@ -42,9 +42,9 @@ class PagesManagerViewController : UIViewController {
   override func viewDidAppear(animated: Bool) {
     super.viewDidAppear(animated)
     
-    if !UserSettingsManager.UserSetting.DidConfiguredMedicine.getBool() {
-      appDelegate.presentInitialSetupScreen(withDelegate: currentViewController)
-    }
+//    if !UserSettingsManager.UserSetting.DidConfiguredMedicine.getBool() {
```

Comment:
In every future pull-request do not forget to remove unnused code (don't just comment it) ! :)